README.md

```diff
@@ -1,4 +1,4 @@
-# AEPCore
+# Adobe Experience Platform Core SDK
 
 <!--
 on [![Cocoapods](https://img.shields.io/cocoapods/v/AEPCore.svg?color=orange&label=AEPCore&logo=apple&logoColor=white)](https://cocoapods.org/pods/AEPCore)
```

Comment:
when we add the cocoapods badges up here, let's make sure we add them for each of the extensions in this repo